.prettierrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": false,
+  "jsxSingleQuote": false,
```

Comment:
Any reason why you wouldn't want everything to be uniform?

Reply:
We agreed that we preferred single quotes in JavaScript code, but double for HTML. Since JSX is a representation of HTML it made sense to sue double quotes. As for single in JavaScript, I don't know what we have a specific reason for that other than preference. However, I do see that creates an inconsistency within a component file. @kaseyllabelle @cblva Anything arguments you want to add for or against single quotes in JS? Any strong preferences?